roles/common/handlers/main.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: reload-sshd
   service:
-    name: "{{ ssh_service }}"
+    name: sshd
```

Comment:
You made this change on Friday, April 14 in order to complete the deployment. I just included it as part of my WIP PR to log all changes.